v63003/u_inref/gtm8787.csh

```diff
@@ -41,10 +41,11 @@ $gtm_tst/com/getoper.csh "$t"
 # Syslog messages captured in getoper will be logged to syslog.txt
 
 echo "# Searching Sys Log for a KILLBYSIG Error (Expecting nothing, would be found in previous versions)"
-cat syslog.txt |& $grep KILLBYSIG |& $tst_awk '{print $6,$7,$8,$9,$10,$11,$12}'
+cat syslog.txt |& $grep -m 1 KILLBYSIG |& $tst_awk '{print $6,$7,$8,$9,$10,$11,$12}'
```

Comment:
Why is the -m 1 needed here? We anyways do not expect even one KILLBYSIS message.